crates/common/src/evm.rs

```diff
@@ -101,19 +103,6 @@ pub struct EvmArgs {
     #[serde(skip)]
     pub always_use_create_2_factory: bool,
 
-    /// Verbosity of the EVM.
-    ///
-    /// Pass multiple times to increase the verbosity (e.g. -v, -vv, -vvv).
-    ///
-    /// Verbosity levels:
-    /// - 2: Print logs for all tests
-    /// - 3: Print execution traces for failing tests
-    /// - 4: Print execution traces for all tests, and setup traces for failing tests
-    /// - 5: Print execution and setup traces for all tests
-    #[arg(long, short, verbatim_doc_comment, action = ArgAction::Count)]
-    #[serde(skip)]
-    pub verbosity: u8,
```

Comment:
Note: global `verbosity` setting is exact equivalent except the description, not sure what a practical way is to modify the verbosity flag description to be more context dependent